client/Sheepshead/SharedSheep/Deck/Piquet.cs

```diff
@@ -12,8 +14,8 @@ public class Piquet : IDeck
         public Piquet()
         {
             Cards = new List<ICard>();
-            SetAllCards();
-            Shuffle();
+            //SetAllCards();
+            //Shuffle();
```

Comment:
If CardFactory was a static method of Card then all we would have to replace is SetAllCards, everything else could stay the same.